datafoundation/templates/storagecluster.go

```diff
@@ -27,8 +27,9 @@ import (
 // StorageClusterTemplate is the template that serves as the base for the storage clsuter deployed by the operator
 
 const (
-	OSDSizeInTiB            = 4
-	backingStorageClassName = "default-ocs-storage-class"
+	OSDSizeInTiB                  = 1
+	VerticalScalerUpperBoundInTiB = 4
```

Comment:
I am fine with you wanting to change the name of the upper limit of allowed OSD size but

1. OSDSizeInTiB has no use anymore so we should not keep it
2. the new name `VerticalScalerUpperBoundInTiB` need to identify itself as belonging to the OSD value because it is an export of the entire `templates` package. 

maybe:
```suggestion
	OSDSizeUpperBoundInTiB = 4
```

Reply:
> OSDSizeInTiB has no use anymore so we should not keep it
[datafoundation/templates/storagecluster.go#L143](https://github.com/red-hat-storage/managed-fusion-agent/blob/18f7d0634c6e8aa9b27e25c05ed85da1f1219bb8/datafoundation/templates/storagecluster.go#L143)
```Go
"storage": resource.MustParse(fmt.Sprintf("%dTi", OSDSizeInTiB)),
```
As I mentioned this is basically the initialization part.